python/MRzeroCore/phantom/voxel_grid_phantom.py

```diff
@@ -406,39 +438,45 @@ def plot(self) -> None:
             print(f"Plotting slice {s} / {self.PD.shape[2]}")
 
         plt.figure(figsize=(12, 10))
-        plt.subplot(331)
+        plt.subplot(431)

```

Comment:
For many usecases masks are not needed and the plot should not be changed. It also will not work if there are more than 3 masks to plot.

Instead, create and show a second figure _if_ there are any masks (where the subplot count adjusts to the mask count)

Reply:
I have made the suggested changes and now extra rows(s) are added according to the number of masks and whether the `plot_masks` flag is `True` or `False`